### ISSUE TITLE
Documentation on getpage.py and google.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5609]
|<img alt="rahoberg's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| rahoberg reported|
|-|-|
|Trac ID|trac#5609|
|Type|enhancement|
|Created|2012-04-04 01:06:47Z|

Documentation on how to run the Twisted examples google.py and getpage.py at http://twistedmatrix.com/documents/current/web/examples/index.html should explain better what the script is and how to run it.

Attachments:

* [firstpatch.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/0ec/0eccc6abbe8bb88fcdd0015a5b8c83aba71773e1/1025938312606682150d534b3f3968351f472210.patch) (1467 bytes) - added by rahoberg on 2012-04-05 07:45:59Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5609 5609
type__enhancement enhancement
reporter__rahoberg rahoberg
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__documentation documentation
time__1333501607000000 1333501607000000
changetime__1333819517000000 1333819517000000
version__None None
owner__thijs thijs
cc__jesstess cc__jknight cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

`twisted.web.google` is deprecated in [twisted/web/__init__.py](twisted/web/__init__.py) since 11.1 so this example will be removed in the next release and can be ignored for now I think.

**Comment by twisted-trac** [comment]
|<img alt="rahoberg's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|rahoberg removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra set owner to @thijstriemstra|
|-|-|

Thanks for your patch. I'll apply it but use an .misc file instead since it's a small doc enhancement.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [34123]) Apply firstpatch.patch: The google and getpage twisted.web examples now have improved documentation that better explains how to run the script.

Author: rahoberg
Reviewer: thijs
Fixes: [#5609](https://github.com/twisted/twisted/issues/5609)
